plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/database/crud/QueryMethodVisitor.kt

```diff
@@ -0,0 +1,219 @@
+package io.sentry.android.gradle.instrumentation.database.crud
+
+import org.objectweb.asm.Label
+import org.objectweb.asm.MethodVisitor
+import org.objectweb.asm.Opcodes.*
+import java.util.concurrent.atomic.AtomicBoolean
+
+class QueryMethodVisitor(
+    api: Int,
+    methodVisitor: MethodVisitor
+) : MethodVisitor(api, methodVisitor) {
+
+    private val label0 = Label()
+    private val label1 = Label()
+    private val label2 = Label()
+    private val label3 = Label()
+    private val label4 = Label()
+    private val label5 = Label()
+    private val label6 = Label()
+    private val label7 = Label()
+    private val label8 = Label()
+
+    private var instrumenting: AtomicBoolean = AtomicBoolean(false)
```

Comment:
we can omit the type since its inferred and make it a `val` probably

Reply:
Yup, was a leftover from the times when it was just a Boolean 👍